HomeUI/src/views/fluxadmin/ManageFlux.vue

```diff
@@ -308,7 +308,7 @@
             </b-button>
             <confirm-dialog
               target="update-blockedRepositories"
-              confirm-button="Update Blocked Repositories"
```

Comment:
It was correct like it was before, its blocked repositories. Please change it.

Reply:
check other buttons it is too long.. look bad ( big and without space between buttons )
![image](https://github.com/RunOnFlux/flux/assets/62837435/bfc371f3-330d-4b22-8d65-f1f9f1258b43)

Reply:
agree makes sense